doc/manual/src/quick-start.md

```diff
@@ -31,7 +31,7 @@ to subsequent chapters.
 1. Install some packages from the channel:
 
    ```console
-   $ nix-env -iA nixpkgs.hello
+   $ nix-env --install --attr nixpkgs.hello
```

Comment:
The change are good except this and the nix-shell -p line where I wouldn't recognize the commands anymore myself but maybe that's just me.